frontend/public/locales/bg/index.json

```
@@ -22,18 +22,18 @@
     "heading": "Как започна всичко?",
     "content": "736 доброволеца от IT средите, юристи, счетоводители, маркетолози, медици, НПО представители, обединени от идеята да подобрим платформите за дарения в България.",
     "data": {
-      "back-end": "BACKEND (146)",
-      "frontend": "FRONTEND (89)",
-      "full-stack": "FULL-STACK (82)",
+      "back-end": "Backend (146)",
+      "frontend": "Frontend (89)",
+      "full-stack": "Full-Stack (82)",
       "dev-ops": "DEV-OPS (78)",
```

@ani-kalpachka Looks like `DEV-OPS` label is still uppercase, maybe we can make it as `DevOps` to be more easy going

```suggestion
      "dev-ops": "DevOps (78)",
```